qrenderdoc/Windows/PipelineState/VulkanPipelineStateViewer.cpp

```diff
@@ -1444,6 +1444,11 @@ void VulkanPipelineStateViewer::addResourceRow(ShaderReflection *shaderDetails,
           else
             dim = QFormatStr("%1x%2").arg(w).arg(h);
 
+          if(a > 1)
+          {
+            dim += QFormatStr("[%1]").arg(a);
+          }
```

Comment:
I think the previous code is correct in omitting the array dimension for a cubemap, since while it's a 6-slice 2D array in implementation it's logically treated as a single object. i.e. you can't have a 5-slice cubemap or a 7-slice cubemap the size is implicit by definition.